.pre-commit-config.yaml

```diff
@@ -238,6 +238,7 @@ repos:
           ^scripts/.*\.py$|
           ^dev|
           ^provider_packages|
+          ^docker_tests|
```

Comment:
Why we do not want to follow pydocstyle here?

Reply:
pydocstyle is disabled for all tests.

Reply:
When I turned it on, we got the following error messages. This is not what we expect.
```
docker_tests/ci_image.py:28 in public class `TestFiles`:
        D101: Missing docstring in public class
docker_tests/ci_image.py:33 in public class `TestPythonPackages`:
        D101: Missing docstring in public class
docker_tests/prod_image.py:37 in public class `TestCommands`:
        D101: Missing docstring in public class
docker_tests/prod_image.py:72 in public class `TestPythonPackages`:
        D101: Missing docstring in public class
docker_tests/prod_image.py:169 in public class `TestExecuteAsRoot`:
        D101: Missing docstring in public class
```

Reply:
Ah yeah. I miseed the first line wiht `^tests/` - it was hidden in GH interface :)